docs/docs/Tutorials/ROS2/PythonLinearDamperExample.md

```diff
@@ -0,0 +1,286 @@
+```
+
+### Control Policy Target
+
+To implement the torque control control policy, we use the `target` function in `ControlPolicy`.
+This is where we accept feedback data and return a command value. In this case, we need the motor
+`rpm`, and the gains applied to the winding current damping, `scale_factor` and `retract_factor`.
+Typical values for these gains are
+
+- scale_factor = 1
+- retract_factor = 0.6
+
+``` py linenums="52" title="mbari_wec_linear_damper_py/controller.py"
+    def target(self, rpm, scale_factor, retract_factor):
+        """Calculate target value from feedback inputs."""
+        N = abs(rpm)
+        I = self.windcurr_interp1d(N)
+
+        # Apply damping gain
+        I *= scale_factor
+
+        # Hysteresis due to gravity assist
+        if rpm > 0.0:
+            I *= -retract_factor
+
+        return float(I)
+```
+
+So, as you can see we apply a positive damping torque when `N` is negative (piston extending), and
+a positive damping torque when `N` is positive (piston retracting). The damping torque required is
+reduced when retracting.
+
+### Controller
+
+All that is left is to connect the necessary feedback data to the `ControlPolicy`. In this case,
+`rpm`, `scale`, and `retract` are present in `buoy_interfaces.msg.PCRecord` on the `/power_data`
+topic published by the Power Controller running on the buoy.
+
+To access the data, all that is required is to define the callback `def power_callback(self, data)`
+in the `Controller` class, and pass the data to `self.policy.target` to get the desired winding
+current command. Various commands are available, and this time we will be using  
+`self.send_pc_wind_curr_command(wind_curr, blocking=False)`
+
+``` py linenums="107" title="mbari_wec_linear_damper_py/controller.py"
+    def power_callback(self, data):
+        """Provide feedback of '/power_data' topic from Power Controller."""
+        # Update class variables, get control policy target, send commands, etc.
+        wind_curr = self.policy.target(data.rpm, data.scale, data.retract)
+
+        self.get_logger().info('WindingCurrent:' +
+                               f' f({data.rpm:.02f}, {data.scale:.02f}, {data.retract:.02f})' +
+                               f' = {wind_curr:.02f}')
+
+        self.send_pc_wind_curr_command(wind_curr, blocking=False)
+```
+
+Finally, let's set the Power Controller's publish rate to the maximum of 50Hz. Uncomment the line
+to set the PC Pack Rate in `Controller.__init__`:
+
+``` py linenums="79" title="mbari_wec_linear_damper_py/controller.py"
+    def __init__(self):
+        super().__init__('linear_damper')
+
+        self.policy = ControlPolicy()
+        self.set_params()
+
+        # set packet rates from controllers here
+        # controller defaults to publishing feedback @ 10Hz
+        # call these to set rate to 50Hz or provide argument for specific rate
+        self.set_pc_pack_rate_param()  # set PC feedback publish rate to 50Hz
+```
+
```

Comment:
Maybe a note here to remind updating controller names and other updates according to the previous tutorial